NEWS

```diff
@@ -8,6 +8,7 @@ Enhancements
  - Appointment color and importance work now between Outlooks
 
 Bug fixes
+ - Fix reception of calendar event invitations on optional attendees
```

Comment:
It was hard but I'm with @jacalvo this is a bug fix, from the code pov he was suppose to sent the invitations but it wasn't doing so for the "cc", so bug fix, no enhancement.